server/app.ts

```diff
@@ -51,10 +54,8 @@ helpersInitialized.then(
       staticFilesAdminPort,
       staticFilesClientPort,
       proxy,
-      redirectIfApiDomain,
```

Comment:
@ballPointPenguin I _think_ you're right that this won't affect anything, but were you able to verify that there weren't any requests being made through `api.*`? Very unclear to me why this was even here, since AFAICT from looking at the network tab, all the requests hit `pol.is/api/v3`, and none to `api.pol.is`. Let's drop for now, and we can always revisit if we find issues. Assuming everything worked on this branch for you though?

Reply:
I did not have any issues when running this branch. I certainly did not hit every API call. But I don't see anywhere that "api.pol.is" would be used.